File: src/main/java/net/atos/client/sd/model/UnattendedResponse.java
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2023-2024 Lifely
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.client.sd.model;

import java.util.List;

import jakarta.json.bind.annotation.JsonbProperty;

public class UnattendedResponse {

  @JsonbProperty("file")
  public List<File> files;
}

File: src/main/kotlin/net/atos/client/sd/model/wizard/UnattendedResponse.kt
/*
 * SPDX-FileCopyrightText: 2024 Lifely
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.client.sd.model.wizard

import jakarta.json.bind.annotation.JsonbProperty

class UnattendedResponse {
    @JsonbProperty("file")
    var files: List<File>? = null
}
